homework-5/Homework-5.playground/Contents.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+var country: [String: String]
```

Comment:
It's better to call this variable with plural word (`coutries`).

We need to store a continent and a fact in different elements not in the same `String`

Reply:
i.e
var country: [String]
"Germany",
...
var continent: [String]
"Western Europe. The surname Hitler is banned in Germany",
...
And work with them ?

Reply:
No, it's not the best solution here. We need that a continent and a fact will be related to the country. The best option would be ```var countries: [String: (String, String)]

Reply:
okey